### PR TITLE
Oraganize .pcap files.

### PR DESCRIPTION
**Pull request by ChakshuGupta**
It is "In Progress" right now.

Issue:  #122

* [x] add pcap files to database after they are closed ( or finished)
* [x] add to api
* [x] add to web UI (explorer) + download ability

### CODE REVIEW

**Comment by Ali-Razmjoo** [comment]
@ChakshuGupta i'd like to add if we can store pcap file in database? since in contains data and if we want to keep all data in same db it can come handy, what do you think?

File: config.py [hunk 1]
```diff
@@ -57,7 +57,8 @@ def network_configuration():
         "real_machine_identifier_name": "stockholm_server_1",  # can be anything e.g. real_machine_ip_address, name, etc
         "ignore_real_machine_ip_addresses": list(set([real_machine_ip_address, "127.0.0.1"])),
         # e.g. ["10.0.0.1", "192.168.1.1"]
-        "ignore_real_machine_ports": []  # e.g. [22, 80, 5000]
+        "ignore_real_machine_ports": [],  # e.g. [22, 80, 5000]
+        "timeout": 3600 # Default value
```
Comment (Ali-Razmjoo) [inline config.py:61]: be more clear than `timeout`, maybe `split_pcap_file_timeout` or something similar.

File: core/load.py [hunk 2]
```diff
@@ -578,6 +578,24 @@ def stop_modules_processors(configuration):
...
+    Args:
+        argv_options
+
+    Returns:
+        network_config
+    """
+    network_config = network_configuration()
+    # Set the values of the network configuration based on CLI input
+    network_config["store_network_captured_files"] = argv_options.store_pcap
+    network_config["timeout"] = argv_options.timeout_value
```
Comment (Ali-Razmjoo) [inline core/load.py:594]: same.

File: core/load.py [hunk 3]
```diff
@@ -620,6 +638,14 @@ def argv_parser():
     engineOpt.add_argument("--start-api-server", action="store_true",
                            dest="start_api_server", default=False,
                            help="start API server")
+    # Store Network captured files
+    engineOpt.add_argument("--store-pcap", action="store_true",
+                           dest="store_pcap", default=False,
+                           help="store network traffic as pcap files")
+    # Set Timeout value for splitting network captured files
+    engineOpt.add_argument("--timeout", type=int,
```
Comment (Ali-Razmjoo) [inline core/load.py:646]: same.

File: config.py (hunk 1 above)
Comment (ChakshuGupta) [reply config.py:61]: Okay

**Comment by ChakshuGupta** [comment]
@Ali-Razmjoo That sounds like a good idea. I'll look into it.

**Comment by Ali-Razmjoo** [comment]
#117 write tests + documents

**Pull request merged**